.github/workflows/ci.yml

```diff
@@ -271,6 +271,22 @@ jobs:
       uses: actions/checkout@master
       with:
         fetch-depth: 1
+    - name: Set properties file with release tag
+      if: success() && startsWith(github.ref, 'refs/tags/')
+      run: |
+        echo git-sha=${GITHUB_SHA::8} >> properties.yml
+        echo release-tag=${GITHUB_REF/refs\/tags\//} >> properties.yml
```

Comment:
@naturalett Nice! 

Can we swap the `=` for `:`? So that it is `echo git-sha:${GITHUB_SHA::8} >> properties.yml`

Reply:
Done